src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Extensions/RoslynLspExtensions_Position.cs

```diff
@@ -12,4 +14,11 @@ public static LinePosition ToLinePosition(this Position position)
 
     public static string ToDisplayString(this Position position)
         => $"({position.Line}, {position.Character})";
+
+    public static VsLspPosition ToVsLspPosition(this Position position)
```

Comment:
Please don't add this extension method. It is 100% intentional that the `RoslynLspExtensions` and `VsLspExtensions` never cross paths. There is `VsLspFactory` class that can be used at calls sites instead. Just convert your Roslyn LSP Position to a LinePosition and call `VsLspFactory.CreatePosition(LinePosition)`.